app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,52 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+h1,h2,h3,h4 {
+	color: #24C73A;
+}
+
+body {
+	background-color: #fedf6b;
+}
+
+.container-fluid {
+	background-image: url('adventure.jpg');
+	margin: 66px -129px 7px 10px;
+	background-size: cover;
+}
+
+a {
+	color: #0180c0;
+}
+
+h1 {
+	border-bottom: 1px solid #24C73A;
```

Comment:
Color literals like `#24C73A` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#24C73A` should be written as `#24c73a`<br>Line should be indented with spaces, not tabs